cypress/integration/profile.feature

```diff
@@ -0,0 +1,84 @@
+Feature: Profile component
+  I want to change Profile component properties
+
+  Background: Open Profile component page
+    Given I open "Profile" component page
+
+  @positive
+  Scenario Outline: Set email to <email>
+    When I set email to "<email>"
+    Then email is set to "<email>"
+    Examples:
+      | email              |
+      | example@email.com  |
+      | johnsmith@sage.com |
+
+  @positive
+  Scenario Outline: Get avatar via email
+    When I set email to "<email>"
+    Then email is set to "<email>"
+      And avatar is taken from "<avatar>"
+    Examples:
+      | email                | avatar                                                                        |
+      | andrew.tait@sage.com | https://www.gravatar.com/avatar/ec55ecf2e1c7e7e56a904b50245c24a4?s=40&d=blank |
+
+  @negative
+  Scenario Outline: Set email out of scope to <email>
+    When I set email to "<email>"
+    Then email is set to "<email>"
+    Examples:
+      | email                   |
+      | Sample text             |
+      | áéíóú¿¡üñ               |
+      | !@#$%^*()_+-=~[];:.,?{} |
+      | ÄÖÜßäöüß                |
+  # @ignore because of FE-1447
+  # | <> |
+
+  @positive
+  Scenario Outline: Set initials to <initials>
+    When I set initials to "<initials>"
+    Then initials is set to "<result>"
+    Examples:
+      | initials | result                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
+      | A        | data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACgAAAAoCAYAAACM/rhtAAABW0lEQVRYR+2Vv0sDQRCFX5q95q7Qa9RWRAQtA/6obJIiUTBaiH+gpEgiaFIkjZ0IlgomGkuTFMHGNLlG2YUNFpsQ9gVy4Gx53Nx9983Mu0y70/1Bik9GAMnuiEFSIMSgGGQNsPUyg2KQNcDW/58ZHCdjVG8aGA6/cFEqYmN9jZVn6hdmsPP+gdt60zz06CCLw/1sugDvGi18j0ZQSiFJEpyfFRCogIZciEENdl2uYW93B3G8akyeFvPY3tpMB6Bub7N1b2YvjlfMLEZhiJNCbvmAdjk0iW3rw+MTnl9ecXVZMqDMoVvc6w9QqdWRzx1PWuq65gtJA7psuawuBdCCfPYGzvcHStGZSBm02efa2L+bzWQiBaizT8/btGWw2chkojegNaR/adPiZJbheWfSG1Dbab91Zway/YgoCr3/LN6A8xpg7xNAMcgaYOtlBsUga4CtlxkUg6wBtv4Xo3kVGOD+5tAAAAAASUVORK5CYII=                                                                 |
```

Comment:
could we put that huge amount of text to a some sort of data.json file ?

Reply:
done